### PR TITLE
[pipeline](mysqlscan) support mysql scan node

### PR DESCRIPTION
**Pull request by luozenglin**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.75 seconds
 load time: 453 seconds
 storage size: 17123356217 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221208155629_clickbench_pr_60512.html

File: be/src/pipeline/exec/mysql_scan_operator.h [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+namespace doris::pipeline {
+
+class MysqlScanOperatorBuilder : public OperatorBuilder<vectorized::VMysqlScanNode> {
+public:
+    MysqlScanOperatorBuilder(int32_t id, ExecNode* exec_node);
+    bool is_source() const override { return true; }
+    OperatorPtr build_operator() override;
+};
+
+class MysqlScanOperator : public Operator<MysqlScanOperatorBuilder> {
```
Comment (Gabriel39) [inline be/src/pipeline/exec/mysql_scan_operator.h:37]: Please rebase master and replace Operator by SourceOperator

**Review by Gabriel39 — CHANGES_REQUESTED** [review]

**Review by Gabriel39 — COMMENTED** [review]
Please rebase master to pass P1 regression

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Pull request merged**